tests/e2e-tests/src/orchestration.test.ts

```diff
@@ -31,4 +32,9 @@ describe('orchestration', () => {
     expect(response.getContent()).toEqual(expect.any(String));
     expect(response.getFinishReason()).toEqual('stop');
   });
+
+  it('should complete a chat with masking', async () => {
```

Comment:
It might be worth noting somewhere that the test takes a while, do you think a comment would make sense?
At least someone who hasn't run this test will probably be confused why it's running for 20+ seconds.

Reply:
Upon more testing I found that it was just a temporary issue, it is more in line with other LLM requests now, taking about 5-10s which is reasonable considering `gpt-4-32k` is not the fastest of all models 😉 

Most time is actually spent by jest setting up the test (at least on my machine)..